.github/issue_template.md

```diff
@@ -2,6 +2,11 @@
 <!-- (Guidelines for creating a bug report are [available
 here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#creating-a-bug-report)) -->
 
+Paste your executable test case created from one of the scripts found [here](https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html#create-an-executable-test-case) below:
```

Comment:
Should this be the regular 'stable' guides instead?
```suggestion
Paste your executable test case created from one of the scripts found [here](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#create-an-executable-test-case) below:
```

Reply:
The guide linked above is `https://edgeguides.rubyonrails.org/contributing_to_ruby_on_rails.html`. I think it makes more sense to be consistent about linking the same guide rather than having one link to edge and the other link to the stable docs. Also the edge docs link to the reproduction script for both `main` and `stable` issues, so they will still be able to find the latest stable reproduction script template.

That being said, I should change this to a comment to match the above guide reference.

Reply:
That makes sense.